lib/nano.d.ts

```diff
@@ -1269,6 +1273,39 @@ declare namespace nano {
     // Total execution time in milliseconds as measured by the database.
     execution_time_ms: number;
   }
+
+  // http://docs.couchdb.org/en/latest/api/database/find.html#db-index
+  interface CreateIndexRequest {
+    // JSON object describing the index to create
+    index: {
+      // Array of field names following the sort syntax.
+      fields: SortOrder[],
+
+      // A selector to apply to documents at indexing time, creating a partial index.
+      partial_filter_selector?: MangoSelector
+    },
+
+    // Name of the design document in which the index will be created.
+    ddoc?: string
+
+    // Name of the index. If no name is provided, a name will be generated automatically.
+    name?: string,
+
+    // Can be "json" or "text". Defaults to json.
+    type?: 'json' | 'text'
+  }
```

Comment:
Cloudant (and I assume soon CouchDB) has an extra optional, boolean flag here `partitioned` see https://cloud.ibm.com/docs/services/Cloudant?topic=cloudant-query#creating-an-index

Reply:
Okey, i added the field :)